### ISSUE TITLE
Getting `Actor already started` panics using `SystemService::from_registry()` in test code

### ISSUE THREAD

**Issue by vincentdephily** [actix/actix#248]
I'm trying to migrate from 0.7.10 to 0.8.0 and keep getting error of the form    
```rust
thread 'actix-rt:worker:2' panicked at 'Actor already started',
/home/work/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-0.8.0/src/registry.rs:315:21
```

It seems to correspond with starting a system service. I have different `Actors`  on separate `Arbiters` that send messages to the same SystemService.  With 0.7.10   I   used  to    write
```rust
System::current().registry().get::<MyActor>().do_send(msg);
```
And  on 0.8.0 I   switched to
```rust
MyActor::from_registry().do_send(msg);
```

**Comment by fafhrd91** [comment]
fixed in 0.8.1 release

**Comment by theronic** [comment]
`MyActor::from_registry().do_send(msg);` throws Illegal instruction on ARM using actix v0.8.3.

https://github.com/actix/actix/issues/272
